### DISCUSSION TITLE
ASReview in the Cloud

### DISCUSSION THREAD

**Discussion by JonnaLind** [asreview/asreview#1281] [Q&A]
Hi there,

Is there any news on when the Cloud-version will be available?

Kind regards, Jonna Lind

**Comment by J535D165** [comment]
Hi Jonna, we are still working on improving the experience of ASReview in the Cloud. As you might already know, you can run ASReview in a cloud environment (see for example our Docker images); however, there are some limitations. One of these limitations is that ASReview doesn't provide built-in authentication. Therefore you should take care of this yourself. The current version of ASReview also doesn't provide multiuser functionality, i.e., labeling projects together or sharing projects. 

The development team is working on implementing features such as authentication and multi-user workflows. This is a time consuming process, but we are slowly getting there. It's difficult to give a timeline for this at this point as we rely heavily on the availability of the core development team and technical challenges that we may not have anticipated. Building features like this in an open, free, and secure way requires some extra challenges. Developments regarding these new features can be followed in PR https://github.com/asreview/asreview/pull/1168 (highly technical discussions). We expect the first beta version of the new features in the quarter of 2023.

**Comment by JonnaLind** [comment]
Hi Jonathan,

I’m eager to know if there are any developments in the ASReview in the Cloud? Could you please tell me, what’s happening now, and you have something of a timeline for when the cloud version will be available?


Kind regards,
Jonna Lind

**Reply by Rensvandeschoot** [reply]
You might be interested in the GitHub page where all the cloud options are introduced: https://github.com/asreview/cloud-usage

**Reply by JonnaLind** [reply]
This still looks quite 'beta' to me... Is there any news on the work on a cloud solution for end users?

**Reply by Rensvandeschoot** [reply]
It's running in full operation, but we do not offer a SAAS version. If that is what you are looking for, you might want to take a look at other software; see [this overview ](https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text)I created for a comparison.

**Comment by JonnaLind** [comment]
Thanks, Rens,
I’ll present it to my ICT colleagues, this is a bit too technical for me….

**Reply by Rensvandeschoot** [reply]
Indeed  the instructions are for the people who can host this on a server for end-users. You can even contact a company to do this for you.
